CHANGELOG.adoc

```diff
@@ -0,0 +1,275 @@
+= Revision History
+:toc:
+
+[[v1.3.3-sep-1-2016]]
+== 1.3.3 September 1, 2016
```

Comment:
@budhrg I do not think this matter much. As long as we are consistent and can communicate  the time of release, we should be good.